listenbrainz/influx-writer/influx-writer.py

```diff
@@ -191,18 +196,32 @@ def write(self, listen_dicts):
             timestamps = {}
             for result in results.get_points(measurement=get_measurement_name(user_name)):
                 dt = datetime.strptime(result['time'] , "%Y-%m-%dT%H:%M:%SZ")
```

Comment:
do we do stuff with dates in this format from influx often? If so, it'd be nice to have this abstracted away somewhere